### PR TITLE
maint(ct): clean up CSC tests

### PR DESCRIPTION
**Pull request by smartcontracts**
**Description**
Yet another test cleanup commit. Cleans up the ChainStorageContainer
tests with the same techniques as the previous commits along the same
lines.

Last one for the night!

### CODE REVIEW

**Review by Inphi — APPROVED** [review]

File: packages/contracts/test/contracts/L1/rollup/ChainStorageContainer.spec.ts [hunk 1]
```diff
@@ -96,18 +71,14 @@ describe('ChainStorageContainer', () => {
...
           }
         })
 
         for (let i = len - 1; i > 0; i -= Math.max(1, len / 4)) {
           it(`should be able to delete everything after and including the ${i}th/st/rd/whatever element`, async () => {
             await expect(
-              ChainStorageContainer.connect(otherSigner)[
-                'deleteElementsAfterInclusive(uint256)'
-              ](i)
+              ChainStorageContainer['deleteElementsAfterInclusive(uint256)'](i)
```
Comment (maurelian) [inline packages/contracts/test/contracts/L1/rollup/ChainStorageContainer.spec.ts:81]: Did we modify a formatter/linter rule or something? I didn't think we really had the freedom to decide about stuff like this

**Review by maurelian — APPROVED** [review]
I see no change in functionality tested. LGTM

File: packages/contracts/test/contracts/L1/rollup/ChainStorageContainer.spec.ts (hunk 1 above)
Comment (smartcontracts) [reply packages/contracts/test/contracts/L1/rollup/ChainStorageContainer.spec.ts:81]: Hmmm what do you mean by this? What sort of stuff are you referring to?

Comment (maurelian) [reply packages/contracts/test/contracts/L1/rollup/ChainStorageContainer.spec.ts:81]: Shouldn't [running](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts/package.json#L34-L37) `eslint --fix .` make the formatting decision for you?

Comment (smartcontracts) [reply packages/contracts/test/contracts/L1/rollup/ChainStorageContainer.spec.ts:81]: Oh it's because I removed `.connect(otherSigner)`

Comment (smartcontracts) [reply packages/contracts/test/contracts/L1/rollup/ChainStorageContainer.spec.ts:81]: Running `yarn lint:fix` worked for me

**Review by tynes — APPROVED** [review]

File: packages/contracts/test/contracts/L1/rollup/ChainStorageContainer.spec.ts (hunk 1 above)
Comment (tynes) [reply packages/contracts/test/contracts/L1/rollup/ChainStorageContainer.spec.ts:81]: I think its your version of typescript locally

Comment (maurelian) [reply packages/contracts/test/contracts/L1/rollup/ChainStorageContainer.spec.ts:81]: that was it yep

**Pull request merged**